modules/react/common/lib/responsive/responsiveContext.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import {
+  CanvasBreakpoints,
+  PartialEmotionCanvasTheme,
+  useTheme,
+} from '@workday/canvas-kit-react/common';
+
+export interface ResponsiveContextConfig {
+  width: number;
```

Comment:
do we need Jsdocs for these?

Reply:
I would say yes because it's exported. I can add those.

Reply:
I removed the `export` as it's not needed

Reply:
So I don't we need JSDocs for these